### ISSUE TITLE
Well, it seems this repo has not support now

### ISSUE THREAD

**Issue by daniel94104** [akveo/blur-admin#306]
Just checked out the latest commits. It was in Dec 2016.

**Comment by stanleyxu2005** [comment]
Yes. The active development is now moved onto Angular 2 code base. _(However, I don't pretty regard ng2 as a successful replacement, you can see the number of modules in their demo dashboard is much less than ng1 version)_

**Comment by daniel94104** [comment]
I agree. I also heard someone saying that Angular2 is not that stable. So, I am still in angular 1.x, and waiting for later updates.

**Comment by PhilLaFayette** [comment]
I've refactored this project for WebPack3 and Angular 1.6.4.

As SOON as I have the time, I'll upload it to a new branch.

Note: It's kinda hacky; I didn't feel like refactoring ALL the angular code, so I added some gulp build magic. Perhaps I'll get around to this by the weekend. No promises, I'm freakin' SWAMPED at work right now...

**Comment by PhilLaFayette** [comment]
SORRY it took me so long, but here's a nice start: https://github.com/PhilLaFayette/electronics
